lib/ecto/repo.ex

```diff
@@ -56,6 +56,9 @@ defmodule Ecto.Repo do
       use the `:repo` property in the event metadata for distinguishing
       between repos.
 
+    * `:stacktrace`- publishes the stacktrace in telemetry events and 
+      allows more advanced logging.
+
```

Comment:
Its a bit of a gray area because, well, this is how ecto_sql uses it, but it may be used differently elsewhere :~